auctioneerrpc/gen_protos_docker.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
nit: those two files aren't actually used as we compile the `auctioneerrpc` package from the `poolrpc`'s `gen_protos.sh` (see bottom of the file).

Reply:
Ah yeah, I think I added these when I was figuring out where it's generated + accidental commit :') removed